source/simulator/core.cc

```diff
@@ -1847,6 +1855,9 @@ namespace aspect
   template <int dim>
   void Simulator<dim>::run ()
   {
+    walltime.start();
+    walltime_from_resume=0.;
```

Comment:
I think this =0 should be in the initializer list of Simulator.

Reply:
Ok thanks, I moved it to the initializer list but I'm not sure the syntax is correct